apps/emqx/src/emqx_config.erl

```diff
@@ -280,7 +281,7 @@ init_load(SchemaMod, RawConf) when is_map(RawConf) ->
                             maps:with(RootNames, RawConfWithEnvs)).
 
 include_dirs() ->
-    [filename:join(emqx:data_dir(), "configs")].
+    [filename:join(emqx:data_dir(), "configs") ++ "/"].
```

Comment:
Is this an issue of hocon_schema? If so we may also need to fix it in the hocon lib.

Reply:
Fix on https://github.com/emqx/hocon/pull/166